other/Map tools/Unova.tmx

```diff
@@ -14,37 +14,45 @@
   </data>
  </layer>
  <layer id="6" name="Towns" width="100" height="60">
   <data encoding="base64" compression="zstd">
-   KLUv/WDAXE0OAJJNIiMgFwICQ1V1YhGLJLABCTEGmOKLH1BGebEYiAAbTFAzeXpzBwN5H8/jdQwI/ASM42286WlM0LSMh/GkF0mQtItn8SoeBRR1fgByHs67eTavhubNPJkX82BenstreSxvReWlPJR38kwWLoUr4UJYySN5I0/khUxcElfEBXFwGVwFF8F6uByuhovhgIuoYWsJeTQGAAMBCCSYgQ0SQBFAAIFAMECgEfwVwSRBhUEkzHLqDhWdvrL6VlIPNC1/cOcFpYSbanhTCxfVIU1qy59td0FVhjQd4QWssM6K1Qe5rQypBG13HEcc1pYJAwhj0vKYOKw9zCnu1y32luvEYTGEUEZjLdvLvK8bge3yOnAJCXxCBiLhgafG4YMQU7Mth08zb2ASxsAFNjgH2tQdSYszcGGMz4EidUfS4hy4MOJzoFLa7IUSy4GCd0dw92qnBtrm6zaXt4nSxuFFC3+uapt8lF9AGPHkmJzve+EjT20DG+0L2DyfLI/s4LwXjq5rS804v4Di+eTvSAwOe+HgorbWjPYFbHlyyP4YfPBDsNqG2PUFFJUDxts+5hm8/3yg8kVUvskpYkFahaMmkx5HMAFO5uSH8Mu0BBo=
+   KLUv/WDAXLURACbeRTHg8gggAPNJ0VdIAKSAA8AhvBMWDRQkUAAUJIS8VlVVzv/V/kMzMAMzMAMrJfBHNGoHNAA5ADgAN/Nk3oQn4W08jTd6ohfhQYh5MO/lPXgOXoPH4C14Cl6Ch+BcXstjeQeegVfgEXihB3qfA8nj1HHi0Hk5D+dteBpehofhjTyRF/JABJwCnACcMm6ezat5F56F9/E8XsfjeBUehbRx0jiNTiKaATOXTIwH4714I56IF+KBeCtP5aU8lNAJdPqcBpSJq8QVc8FcLz2P5+28D8/D6/A4vJNn8koeyccBReJyuVoullq5VK6UC6VOLpOr5CKpkUvkCrlAXFwtrp6L59phca24dK6cC0fFleK6uWyuGhTXiYvmCoCAqEEHyM9QGA0HUINEBSx0ABKgEPwJAoXgjxAMIqhQiINB6L0BFU+rUG/leqFp+YPuvHYaXgrhVWqq6vSm2vJHdhf9yvQmETJspXqsTG+oD7ItCthx3nE4UuaHGbuk5ajjMAJhTrm8btlbHigO/0GM2cbK7Jzy+Lo5MC7vwEkgsAQNIKEAU2H4IHJqthvhaZkPIKEPXNhNOVCW3ZGkGAIXdJwcKGF3JCvGwAU3lANdQ9kLvzcHyqI7ZrhmoGP60bbWtl1tkMrcwjP22kSNT88tPGDUhupEOrfwCWdtmkekcwsfeGrjdqZ84a44tbmY1ReSgMl3M5/izDe5RVySpwAoK0W2PmDKYgRFseJBkVZRULQUsslSL2a/bZo/
   </data>
  </layer>
  <layer id="7" name="Dungeons" width="100" height="60">
   <data encoding="base64" compression="zstd">
-   KLUv/WDAXEUGAPAAAHAPAABxcg94DwAAeXoPgA8AAIEPAAB8DwAAgg9FqOCFRR8wgwmSBKIDEphC8A0CDMFrBBrBNwQgEFJeuAdBL0yv6CKLKp6LS1X8MSUJ2dXlSym6sVNyRUyX5a/Z2mtN/Kv5zYh7bca/mt8Mstdqs2KcmZif0DmO1t248OwAJzCloYGJqqJmlTXWbnVJVJHFgPoWLdJoqW0ZWnHYiwvc6OlUyfGkGzUykq5frB/7ZrT97nAzaINtMePT7DYYEzMlbf4Eb5MC
+   KLUv/WDAXAUIANQDAABwDwAAcXIPAHgPAAB5eoAPAACBDwAAfA8AAIKBgYFtDwAAbg8AAG91DwAAdg8AAHd9DwAAfg8AAH+BfEiokAlBmgDgAglngFAHEqAQZFAEKoFFsEPw/ywBikitB4EwGxVbPOwiF//i8hQfoxXMveR6Bhm9Bn9d5vQRo5ucXi759H1GfT99It+k7V389L2mvphxunZb7pTTu/a1pgu5VaEWdBXMNC6cVtii/aPMfRTPXUH0fwNx687HJh/gZOTkTQGP4DsU8Aiyj3f0poTNHxTDQO67cfT94cEbVhoDVMPAXEYhF6LDAHIHyFEZApYy4OT/5IXuxwp/9hHQ9Ac=
   </data>
  </layer>
+ <layer id="12" name="Decor" width="100" height="60">
+  <data encoding="base64" compression="zstd">
+   KLUv/WDAXOUAAGgAAM0GAADOBgAAzwYAAwCZmFw+EASqf5x2gAU=
+  </data>
+ </layer>
+ <imagelayer id="13" name="base" visible="0">
+  <image source="../../../../../Downloads/unovabase.png" width="1600" height="960"/>
+ </imagelayer>
```

Comment:
Could you remove this layer, as it will be a broken reference for anyone else working on the maps.

Reply:
Removed